src/core/public/chrome/nav_links/nav_links_service.ts

```diff
@@ -53,6 +53,13 @@ export interface ChromeNavLinks {
    */
   getNavLinks$(): Observable<Array<Readonly<ChromeNavLink>>>;
 
+  /**
+   * Get an observable for the current link updaters. Link updater is used to modify the
+   * nav links, for example, filter the nav links or update a specific nav link's properties.
+   * {@link LinksUpdater}
+   */
+  getLinkUpdaters$(): BehaviorSubject<LinksUpdater[]>;
```

Comment:
should we update the doc https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/chrome/README.md 
and https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/public.api.md as well?

Reply:
good point, I will update [README.md](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/chrome/README.md), `public.api.md` is automatically generated.